src/Test.FunctionalTests.BinSkim.Driver/AnalyzeCommandTests.cs

```diff
@@ -127,6 +127,34 @@ public void AnalyzeCommand_DeterminismTest()
             log.Runs[0].Invocations[0].ToolConfigurationNotifications.Count(t => t.Message.Text.Contains("E_PDB_NOT_FOUND")).Should().Be(1);
         }
 
+        [Fact]
+        public void AnalyzeCommand_ZeroByteTest()
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

The test binaries are existing ones.
The reason I added one empty "WORKSPACE" file with 
one PE and one elf is,
if there is no valid file at all the output will be changed to no valid files instead, and we don't want that, so with them this test can run in both Windows and Linux.

Reply:
To make sure I understand this correctly, @shaopeng-gh you added 0-byte files, correct? Could we not do this by extending the `IFileSystem` mock, instead of adding additional files?